tests/components/opnsense/test_device_tracker.py

```diff
@@ -1,41 +1,42 @@
 """The tests for the opnsense device tracker platform."""
-from homeassistant.setup import async_setup_component
 
+import unittest
+from unittest import mock
+
+from homeassistant.components import opnsense
 from homeassistant.components.opnsense import CONF_API_SECRET, DOMAIN, OPNSENSE_DATA
 from homeassistant.const import CONF_URL, CONF_API_KEY, CONF_VERIFY_SSL
+from homeassistant.setup import setup_component
 
-from tests.common import MockDependency, mock_coro_func
+from tests.common import get_test_home_assistant
 
-FAKEFILE = None
 
-VALID_CONFIG_ROUTER_SSH = {
-    DOMAIN: {
-        CONF_URL: "https://fakehost",
-        CONF_API_KEY: "fake_key",
-        CONF_API_SECRET: "fake_secret",
-        CONF_VERIFY_SSL: False,
-    }
-}
+class TestOpnSenseDeviceTrackerSetup(unittest.TestCase):
```

Comment:
The tests should still be standalone coroutine pytest test functions. Please just change the mocking as requested.

Reply:
Done in: https://github.com/home-assistant/home-assistant/pull/26834/commits/4f7c26932e865946feb6a812bc4f2d5ea6caf206